recipes/neurdflib/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "neurdflib" %}
+{% set version = "5.0.0.post1" %}
+{% set sha256 = "65ef138f7a20f646b5257d312e14c624a83abdac4df234e5edcc77dc63bef5e1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
If this is a pure python package you can also add 
```
noarch: python
```

inside of the ``build`` section

Reply:
It is indeed. Thank you for the suggestion!